src/stores/currency-update.md

```diff
@@ -23,18 +23,18 @@ _Updating Currency Rates_
 
 1. Set **Import Service** to the currency rate provider.
 
-   WebserviceX is the default provider.
+   Fixer.io is the default provider.
```

Comment:
this one is odd.   if fixer should be lower case, maybe rework the sentence as The default provider is fixer.io.